source/_integrations/google_assistant_sdk.markdown

```diff
@@ -87,6 +87,44 @@ The integration setup will next give you instructions to enter the [Application
 
 {% enddetails %}
 
+## Enable personal results
+
+If you want to enable personal commands (e.g. "remind me tomorrow at 9pm to take out trash") follow below steps after successfully installing the integration. This is for advanced users since it requires creating an OAuth client ID of Desktop app, running a Python program on your desktop (or laptop), and copying the resulting credentials to your Home Assistant config directory.
+
+{% details "Create credentials file" %}
+
+1. Navigate to https://console.cloud.google.com/apis/credentials
+2. Select the project you created earlier from the dropdown menu in the upper left corner.
+3. Click Create credentials (at the top of the screen), then select OAuth client ID.
+4. Set the Application type to "Desktop app" and give this credential set a name (like "Home Assistant Desktop Credentials").
+5. Click Create.
+6. In the OAuth client created screen, click on Download JSON.
+7. Rename the downloaded file to `client_secret.json`
+8. On your Windows or Linux or Mac machine download Python if you don't have it already.
+9. Open terminal (on windows click on start and then type cmd).
+10. On the terminal run the following commands (preferably in a Python virtual environment): 
+11. `python -m pip install --upgrade google-auth-oauthlib[tool]`
```

Comment:
I think this might be a bit overboard for the average user. How would they even do that?

Reply:
Agreed. That's the whole section starts with "This is for advanced users". I changed the heading to "Enable personal results (for advanced users)" to make it more clear.